### ISSUE TITLE
Background image coming from on Avatar 

### ISSUE THREAD

**Issue by Amitdygon** [SceneView/sceneform-android#444]
Hello Support 

I want to change my   com.google.ar.sceneform.SceneView background when i am changes the background color using this  ` binding.transparentSceneView.setBackgroundColor(Color.BLUE)  ` its working. Here i have attached the screen shot.
![image](https://user-images.githubusercontent.com/45029280/232719580-ffb62ab9-6343-4682-adbc-13bbc603d568.png)

But when I have added the background resource its not showing the 3d Model Here i have attched the screen shot i have checked its showing in the front of 3d Model. 
![image](https://user-images.githubusercontent.com/45029280/232721325-82d6ab15-4c40-46ef-bc7f-4598af9272ae.png)

I have looked some other solution and create a new node and set into ar scence view and its looking good but its have a issue while rotating the models the background is also rotated with 3d Model here i have attched the  Video and code for this.

  Create a viewRender 
`ViewRenderable.builder().setView(requireContext(), R.layout.layout_image_back)
            .build().thenAccept {
                viewRenderable = it
            }.exceptionally {
                showToastLong(it.message ?: "")
                null
            }`

` private fun addNodeToScene(profileModel: ModelRenderable?) {
        clearAllModel()

        val transformationSystem = makeTransformationSystem()
        val tempNode = Node()
        tempNode.renderable = profileModel

        val collisionShape: Box = tempNode.collisionShape as Box
        val radius = collisionShape.size.x

        val dragTransformableNode = DragTransformableNode(radius, transformationSystem).apply {
            parent = binding.transparentSceneView.scene
            translationController.isEnabled = false
            scaleController.isEnabled = true
            rotationController.isEnabled = true
            scaleController.minScale = 0.86f
            localPosition = Vector3(0f, -1f, 0f)
            renderable = profileModel
        }

        binding.progressBar.gone()

        val filament = dragTransformableNode.renderableInstance?.filamentAsset
//        val renderable = binding.transparentSceneView.scene.renderer?.filamentRenderer?.engine?.renderableManager

        // LeftArm ->155  RightArm -> 174

        filament?.releaseSourceData()
        val leftEntity= filament?.getFirstEntityByName("LeftArm")?.let {
            val defaultTransform = it.getTransform()
            val transform =
                rotation(Quaternion(1F, 0f, 0f, Math.PI.toFloat()))
            it.setTransform(transform)
        }
        val rightEntity = filament?.getFirstEntityByName("RightArm")?.let {
            val defaultTransform = it.getTransform()
            val transform =
                rotation(Quaternion(1F, 0F, 0F, Math.PI.toFloat()))
            it.setTransform(transform)
        }
        filament?.animator?.updateBoneMatrices()

        binding.transparentSceneView.scene.addChild(dragTransformableNode)
        // this is for add image into background
           Node().apply {
               parent = binding.transparentSceneView.scene
               viewRenderable?.sizer = FixedWidthViewSizer(2F)
               renderable = viewRenderable
               isEnabled = false
               localScale = Vector3(2f, 2f, 2f)
               localPosition = Vector3(0f, -1f, -0.3f)
               binding.transparentSceneView.scene.addChild(this)
               isSelectable = false

           }
        dragTransformableNode.select()
        binding.transparentSceneView.scene.addOnPeekTouchListener { hitTestResult: HitTestResult?, motionEvent: MotionEvent? ->
            transformationSystem.onTouch(
                hitTestResult,
                motionEvent
            )
        }

//        binding.transparentSceneView.setBackgroundColor(Color.BLUE)
//        binding.transparentSceneView.setBackgroundFromUrl("https://preprod-images.avatus.com/Avatus/IMG_20221104_190859542_1667569140802.jpg")

    }

    private fun Int.getTransform(): Mat4 {
        val tm =
            binding.transparentSceneView.scene.renderer?.filamentRenderer?.engine?.transformManager
        val arr = FloatArray(16)
        tm?.getInstance(this)?.let { tm.getTransform(it, arr) }
        return Mat4.of(*arr)
    }

    private fun Int.setTransform(mat: Mat4) {
        val tm =
            binding.transparentSceneView.scene.renderer?.filamentRenderer?.engine?.transformManager
        tm?.getInstance(this)?.let { tm.setTransform(it, mat.toFloatArray()) }
    }

    private fun clearAllModel() {
        val children: List<Node> =
            java.util.ArrayList<Node>(binding.transparentSceneView.scene.children)
        for (node in children) {
            if (node is AnchorNode) {
                if (node.anchor != null) {
                    node.anchor?.detach()
                }
            }
        }
    }

    private fun makeTransformationSystem(): TransformationSystem {
        val footprintSelectionVisualizer = FootprintSelectionVisualizer()
        return TransformationSystem(
            activity?.resources?.displayMetrics,
            footprintSelectionVisualizer
        )
    }`

Here i have attached the Video.

https://user-images.githubusercontent.com/45029280/232723970-c3d7f570-f027-45b6-a9fc-bf1d4b35f512.mp4

Please help me to fixed this I can use` binding.transparentSceneView.setZOrderOnTop(true)` beacuse when i used this its show front of all the screen.

**Comment by SamihMustafa** [comment]
if there is no UI on top of the surfaceView, you can follow this code from filament. https://github.com/google/filament/tree/main/android/samples#transparent-rendering

**Comment by Amitdygon** [comment]
> if there is no UI on top of the surfaceView, you can follow this code from filament. https://github.com/google/filament/tree/main/android/samples#transparent-rendering

I am using google ar scence is there any possible way to do this into google ar. not into filament. beacuse i have to record the ar view and if i used two view i have one more challange for record both view

**Comment by SamihMustafa** [comment]
if thats also using surfaceView, you might need setZOrderMediaOverlay

**Comment by Amitdygon** [comment]
> setZOrderMediaOverlay  I have used this but if i used this one its show the model in the front of all view

**Comment by Amitdygon** [comment]
Hello I have done some code and add two nodes at here and only rotate the 1st model and its seem to working alomost 80%.
